static/app/views/replays/replayTable.tsx

```diff
@@ -1,148 +1,188 @@
-                <Duration
-                  seconds={
-                    Math.floor(
-                      dataEntries[replay[idKey]]
-                        ? dataEntries[replay[idKey]]['equation[0]']
-                        : 0
-                    ) || 1
-                  }
-                  exact
-                  abbreviation
-                />
-              </Item>
-              <Item>
-                {dataEntries[replay[idKey]]
-                  ? dataEntries[replay[idKey]]?.count_if_event_type_equals_error
-                  : 0}
-              </Item>
-              <Item>
-                <ReplayHighlight data={dataEntries[replay[idKey]]} />
-              </Item>
-            </React.Fragment>
-          ) : (
-            <React.Fragment>
-              <Item>
-                <Placeholder height="24px" />
-              </Item>
-              <Item>
-                <Placeholder height="24px" />
-              </Item>
-              <Item>
-                <Placeholder height="24px" />
-              </Item>
-            </React.Fragment>
-          )}
-        </Fragment>
+    <StyledPanelTable
+      isLoading={isFetching}
+      isEmpty={replays?.length === 0}
+      headers={[
+        t('Session'),
+        showProjectColumn && minWidthIsSmall ? t('Project') : null,
+        <SortLink
+          key="startedAt"
+          role="columnheader"
+          aria-sort={
+            sort.field === 'startedAt'
+              ? sort.kind === 'asc'
+                ? 'ascending'
+                : 'descending'
+              : 'none'
+          }
+          to={{
+            pathname,
+            query: {
+              ...location.query,
+              sort: sort.kind === 'desc' ? 'startedAt' : '-startedAt',
+            },
+          }}
+        >
+          {t('Start Time')} {sort.field.endsWith('startedAt') && sortArrow}
+        </SortLink>,
+        <SortLink
+          key="duration"
+          role="columnheader"
+          aria-sort={
+            sort.field.endsWith('duration')
+              ? sort.kind === 'asc'
+                ? 'ascending'
+                : 'descending'
+              : 'none'
+          }
+          to={{
+            pathname,
+            query: {
+              ...location.query,
+              sort: sort.kind === 'desc' ? 'duration' : '-duration',
+            },
+          }}
+        >
+          {t('Duration')} {sort.field === 'duration' && sortArrow}
+        </SortLink>,
+        t('Errors'),
+        t('Interest'),
+      ]}
+    >
+      {replays?.map(replay => (
+        <ReplayTableRow
+          key={replay.id}
+          replay={replay}
+          organization={organization}
+          showProjectColumn={showProjectColumn}
+          minWidthIsSmall={minWidthIsSmall}
+        />
       ))}
+    </StyledPanelTable>
+  );
+}
+
+function ReplayTableRow({
+  minWidthIsSmall,
+  organization,
+  replay,
+  showProjectColumn,
+}: RowProps) {
+  const {projects} = useProjects();
```

Comment:
It might be better to pass `useProjects` the projectId so that it fetches the project if it is not already cached.

Reply:
Lets follow up on this one. 

It looks like the hook takes slugs, not ids afaik.

also it has this message at the top:
```
* NOTE: Currently ALL projects are always loaded, but this hook is designed
 * for future-compat in a world where we do _not_ load all projects.
```

so we're safe _for now_